shell/app/styles/antd-extension.scss

```diff
@@ -417,6 +417,14 @@ div.ant-table {
   }
 }
 
+.ant-table .ant-form-item-has-error {
+  margin: 10px 0 !important;
+
+  .ant-form-item-explain-error {
+    font-size: 12px;
+  }
+}
+
```

Comment:
![2021-08-09 10 32 40](https://user-images.githubusercontent.com/3955437/128655197-af577ce1-5f51-4db6-a22d-388e0a920c5a.gif)
try to use official interaction.